templates/taxi/car_detail.html

```diff
@@ -1,22 +1,33 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>{{ car.model }}</h1>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    Drivers
+    {% if car in user.cars.all %}
+      <a href="{% url 'taxi:reverse-driver' pk=car.id %}" class="btn btn-danger link-to-page">
+        Delete me from this car
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:reverse-driver' pk=car.id %}" class="btn btn-success link-to-page">
+        Assign me to this car
+      </a>
+    {% endif %}
   </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
   <hr>
-  <ul>
-    {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-    {% endfor %}
-  </ul>
+  {% for driver in car.drivers.all %}
+    <strong>Username:</strong> {{ driver.username }}
+    <br>
+    <strong>Full name:</strong> {{ driver.first_name }} {{ driver.last_name }}
+    <br>
+    <strong>Licence number:</strong> {{ driver.license_number }}
+    <hr>
+  {% endfor %}
+  <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
+    Delete
+  </a>
+  <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
+    Update
```

Comment:
<img width="1387" alt="image" src="https://user-images.githubusercontent.com/95870462/200531809-280d0c79-da81-48a8-93ca-c7c1989994de.png">
in our example, the update and delete buttons at the bottom left. I think it is more user friendly.

Reply:
Ok
![image](https://user-images.githubusercontent.com/103896482/200535041-73f8ff4d-6734-4f0c-a0fc-fe4dce508a10.png)